packages/account-v2/src/modules/ManualForm/manualFormInputs.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react';
+import { Field, FieldProps } from 'formik';
+import { useSettings } from '@deriv/api';
+import { WalletDatePicker } from '../../components/base/WalletDatePicker';
+import { WalletText } from '../../components/base/WalletText';
+import { WalletTextField } from '../../components/base/WalletTextField';
+import { TManualDocumentTypes } from '../../constants/manualFormConstants';
+import { getFieldsConfig, getTitleForFormInputs } from '../../utils/manualFormUtils';
+
+type TManualFormInputsProps = { selectedDocument: TManualDocumentTypes };
+
+export const ManualFormInputs = ({ selectedDocument }: TManualFormInputsProps) => {
+    const fieldsConfig = getFieldsConfig(selectedDocument);
+    const { data: settings } = useSettings();
+    const isExpiryDateRequired = settings?.country !== 'ng';
```

Comment:
I suggest moving business logic to a hook instead of keeping in all over the code in the modules.

Reply:
sure 👍🏼